index.html

```diff
@@ -233,9 +233,9 @@ <h3 id="text19" class="style6">Sonia Singla</h3>
                                             alt="Apprentice Alembic - Crypto Coven NFT" /></div>
                                     <h3 id="text34" class="style4">Co-Founder</h3>
                                     <h3 id="text19" class="style6">Spoorthi Satheesha</h3>
-                                    <p id="text20" class="style5">Blockchain Protocol Engineer who has been working with
-                                        ⚛️ Cosmos ecosystem for over a year.
-                                        Loves knitting, DotA2, Overwatch and ranting about TV shows.
+                                    <p id="text20" class="style5">Blockchain Protocol Engineer working in the
+                                        ⚛️ Cosmos ecosystem since 2021.

```

Comment:
@spoo-bar  using "over a year" will quickly get out of date, wdyt about providing a year? You can add the month if you want to stake a starting point